index.js

```diff
@@ -1,38 +1,80 @@
 const HOME_MENU_BUTTONS_ARRAY = [
     {
-        id: "internet-contents",
-        innerText: "Internet Contents"
+        id: "internetcontents-tab",
```

Comment:
Here is some problem with naming of this id

Reply:
I thought you going to notice it, if I put a - between the two words, then the handleOptionsElementOnClick function is not working. Because it won't end with tab since the dash is there.

Reply:
Solved